thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -3,54 +3,359 @@
  * @module manage/alert/tune
  * @exports manage/alert/tune
  */
-import fetch from 'fetch';
-import Ember from 'ember';
 import Controller from '@ember/controller';
+import moment from 'moment';
+import { buildDateEod } from 'thirdeye-frontend/helpers/utils';
+import { buildAnomalyStats } from 'thirdeye-frontend/helpers/manage-alert-utils';
 
 export default Controller.extend({
+  /**
+   * Be ready to receive time span for anomalies via query params
+   */
+  queryParams: ['duration', 'startDate', 'endDate'],
+  duration: null,
+  startDate: null,
+  endDate: null,
+
+  /**
+   * Set initial view values
+   */
+  filterBy: 'All',
+  isGraphReady: false,
+  isTunePreviewActive: false,
+  isTuneSaveSuccess: false,
+  isTuneSaveFailure: false,
+  selectedSeverityOption: 'Weight',
+  selectedTunePattern: 'None',
+  customPercentChange: '30',
+  selectedTuneType: 'current',
+  customMttdChange: '5',
+  predefinedRanges: {},
+  today: moment(),
+  selectedSortMode: '',
+  sortColumnStartUp: false,
+  sortColumnScoreUp: false,
+  sortColumnChangeUp: false,
+  sortColumnResolutionUp: false,
+
+  /**
+   * Severity display options (power-select) and values
+   */
+  severityMap: {
+    'Weight': 'weight',
+    'Average Change': 'average_change',
```

Comment:
Hi Steve, could you please modify the severity map to the latest version of the wiki, where the mapping is: 
"Percentage of Change" to "weight"
"Absolute Value of Change" to "deviation"
"Site Wide Impact" to "site_wide_impact" 

Thank you so much!

Reply:
Done. Thanks Kexin.